README.md

```diff
@@ -1,26 +1,26 @@
 # tutorial-hello-frontity
 
-This is the project explained in the Step By Step tutorial available at [tutorial.frontity.org](tutorial.frontity.org)
+This is the project as explained in the Step By Step tutorial available at [tutorial.frontity.org](tutorial.frontity.org)
 
-### Accesing to the steps of the tutorial
+### Accessing the steps of the tutorial
 
-Every [commit](https://github.com/frontity-demos/tutorial-hello-frontity/commits/main) of the repo correponds to a step explained in the tutorial. [These commits are tagged](https://github.com/frontity-demos/tutorial-hello-frontity/tags) using the formula "StepX.X"
+Every [commit](https://github.com/frontity-demos/tutorial-hello-frontity/commits/main) in the repo corresponds to a step in the tutorial. [These commits are tagged](https://github.com/frontity-demos/tutorial-hello-frontity/tags) using the formula "StepX.X"
 
-So, once you have cloned your repository by doing (for example)
+So, once you have cloned your repository by running (for example)
 
 ```
-git clone git@github.com:frontity-demos/tutorial-hello-frontity.git
```

Comment:
This command didn't work for me. I've amended it to the command that did work for me.